src/components/Tabs/components/TabMeasurer/TabMeasurer.tsx

```diff
@@ -83,25 +84,31 @@ export default class TabMeasurer extends React.PureComponent<Props, never> {
   };
 
   private handleMeasurement = () => {
-    if (this.containerNode == null) {
-      return;
+    if (this.animationFrame) {
+      cancelAnimationFrame(this.animationFrame);
     }
 
-    const {handleMeasurement} = this.props;
-    const containerWidth = this.containerNode.offsetWidth;
-    const tabMeasurerNode = findDOMNode(this);
-    const hiddenTabNodes =
-      tabMeasurerNode instanceof Element && tabMeasurerNode.children;
-    const hiddenTabNodesArray: HTMLElement[] = [].slice.call(hiddenTabNodes);
-    const hiddenTabWidths = hiddenTabNodesArray.map((node) => {
-      return node.getBoundingClientRect().width;
-    });
-    const disclosureWidth = hiddenTabWidths.pop() as number;
-
-    handleMeasurement({
-      containerWidth,
-      disclosureWidth,
-      hiddenTabWidths,
+    this.animationFrame = requestAnimationFrame(() => {
+      if (this.containerNode == null) {
+        return;
+      }
+
+      const {handleMeasurement} = this.props;
+      const containerWidth = this.containerNode.offsetWidth;
+      const tabMeasurerNode = findDOMNode(this);
+      const hiddenTabNodes =
+        tabMeasurerNode instanceof Element && tabMeasurerNode.children;
+      const hiddenTabNodesArray: HTMLElement[] = [].slice.call(hiddenTabNodes);
+      const hiddenTabWidths = hiddenTabNodesArray.map((node) => {
+        return node.getBoundingClientRect().width;
```

Comment:
if the intent is to get width then I suggest `offsetWidth`.

Reply:
This code already existed.